src/runtime/pkg/katautils/config.go

```diff
@@ -102,6 +102,10 @@ type hypervisor struct {
 	GuestMemoryDumpPath            string          `toml:"guest_memory_dump_path"`
 	SeccompSandbox                 string          `toml:"seccompsandbox"`
 	BlockDeviceAIO                 string          `toml:"block_device_aio"`
+	GuestPreAttestationURI         string          `toml:"guest_pre_attestation_kbs_uri"`
```

Comment:
We can address this in a follow-up, but I think we should ditch the pre-attestation specific AttestationURI and just use one KBS URI field for every runtime class, although there have been some recent AA/CDH updates that are going to make this config more complex.

Reply:
How about just:
```
KbsURI   string   `toml:"kbs_uri"`
```

Reply:
Or:
```
GuestAttestationKbsURI   string   `toml:"guest_attestation_kbs_uri"`
```

Reply:
Well now I am starting to think that it might be a better idea to keep these SEV-specific for now to avoid introducing more complexity to this PR. Then maybe we can make a second PR to standardize between different platforms.

Reply:
Ok, given the emerging plans for a separate config interface (that won't work with SEV-ES), I think we should keep these annotations platform specific.